charts/k3s/templates/coredns.yaml

```diff
@@ -121,6 +121,12 @@ data:
           priorityClassName: "system-cluster-critical"
           serviceAccountName: coredns
           nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 12 }}
```

Comment:
I think we should do `.Values.coredns.nodeSelector` as people might not always want to have the same node selector for vcluster as well as coredns